common/src/main/java/com/wynntils/services/cosmetics/CosmeticsService.java

```diff
@@ -41,7 +41,7 @@ public class CosmeticsService extends Service {
                     RenderLayerParent<AbstractClientPlayer, PlayerModel<AbstractClientPlayer>>,
                     EntityModelSet,
                     WynntilsLayer>
-            ELYTRA_LAYER = (playerRenderer, playerModel) -> new WynntilsElytraLayer(playerRenderer, playerModel);
+            ELYTRA_LAYER = WynntilsElytraLayer::new;
```

Comment:
This is like this so it's in symmetry with the cape layer. revert.

Reply:
i was asking myself that the whole time ^^ thx for the clarifcation